src/Cherry/Stage/Parse/Expression/Literal.elm

```diff
@@ -100,12 +100,17 @@ objectParser expressionParser =
             , separator = ","
             , end = "}"
             , item = 
-                Parser.succeed Tuple.pair
-                    |= Identifier.nameParser
-                    |. Parser.spaces
-                    |. Parser.symbol ":"
-                    |. Parser.spaces
-                    |= expressionParser
+                Parser.oneOf
+                    [ Parser.succeed Tuple.pair
+                        |= Identifier.nameParser
+                        |. Parser.spaces
+                        |. Parser.symbol ":"
+                        |. Parser.spaces
+                        |= expressionParser
+                        |> Parser.backtrackable
```

Comment:
There was a comment somewhere linking to a post about performance when using oneOf with backtrackable. I haven't read it so can't provide useful feedback here, but I'm going to assume you wrote this with that interaction in mind 🙂

Reply:
The performance of `Parser.backtrackable` is pretty awful, yeah, but I'm favouring simplicity/readability over performance for now. The parser will need a rewrite for #2 anyway so I'll probably tackle a lot of the performance issues there.